src/components/MovieCard/MovieCard.scss

```diff
@@ -13,8 +13,9 @@
     justify-content: flex-end;
 
     .menu-icon {
-      padding-top: 14px;
-      padding-right: 14px;
+      margin-top: 14px;
+      margin-left: -14px;
+
       position: absolute;
```

Comment:
Styles for positioning the menu are incorrect, please read more about the `position` property and apply the proper rules. The problems I see are:
1. The menu icon is not visible until it's hovered
2. The menu is positioned with margins
3. The menu has no relative position
4. The menu should be rendered below the menu button (like a regular dropdown)

https://css-tricks.com/almanac/properties/p/position/
https://developer.mozilla.org/en-US/docs/Web/CSS/position

Reply:
All done with small comment to point 3 - relative position is set for movie card element as per private discussion